### PR TITLE
Start rustdoc UI tests when a PR is opened with changes on HTML/CSS/JS

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Fixes https://github.com/rust-lang/rust/issues/60033

**Comment by pietroalbini** [comment]
Seems like nothing is using the new `command` feature?

**Comment by GuillaumeGomez** [comment]
What new `command` feature?

File: highfive/newpr.py [hunk 1]
```diff
@@ -124,13 +124,21 @@ def set_assignee(self, assignee, owner, repo, issue, user, author, to_mention):
...
 
+        commands = {}
         if to_mention and len(to_mention) > 0:
             message = ''
             for mention in to_mention:
                 if len(message) > 0:
                     message += '\n\n'
                 message += "%s\n\ncc %s" % (mention['message'],
                                             ','.join([x for x in mention['reviewers'] if x != user]))
+                cmd = mention.get('command')
```
Comment (pietroalbini) [inline highfive/newpr.py]: Here.

As far as I understood it, the PR allows to add something like `"command": "@bot do-action"` to mentions, and you want to use that to run UI tests whenever rustdoc is changed. You also added a new mention for rustdoc, but there is no trace of `"command": "..."`.

Comment (GuillaumeGomez) [reply highfive/newpr.py]: Oh I see. Good catch!

**Comment by GuillaumeGomez** [comment]
Added the missing commit.

File: highfive/configs/rust-lang/rust.json [hunk 2]
```diff
@@ -38,15 +38,26 @@
...
+            "message": "Some changes occurred in HTML/CSS/JS.",
+            "command": "@highfive: run-doc-ui",
+            "reviewers": ["@GuillaumeGomez"]
+        },
+        "src/librustdoc/html/static/themes": {
+            "message": "Some changes occurred in HTML/CSS themes.",
+            "command": "@highfive: run-doc-ui",
             "reviewers": ["@GuillaumeGomez"]
         },
         "src/src/doc": {
```
Comment (pietroalbini) [inline highfive/configs/rust-lang/rust.json]: Unrelated, but I'm pretty confident this path is wrong. Also, do you want to run the test suite every time the docs are updated?

Comment (GuillaumeGomez) [reply highfive/configs/rust-lang/rust.json]: I just thought about it afterwards: it's pretty useless to run it on this path. Also, the correct path is indeed `src/doc`. I'll fix both.

**Pull request merged**